client/client.js

```diff
@@ -85,15 +87,39 @@ Client.prototype.dataStore = function (message) {
   }
 
   if (changed) {
-    this.lastModified = Date.now();
-    Core.Persistence.persistKey(this.key, this, Client.dataTTL);
+    this._persist();
   }
 
   return true;
 };
 
+Client.prototype.loadData = function (callback) {
+  var self = this;
+
+  // When we touch a client, refresh the TTL
+  Core.Persistence.expire(self.key, Client.dataTTLGet());
+
+  // Update persisted subscriptions/presences
+  Core.Persistence.readKey(self.key, function (clientOld) {
+    if (clientOld) {
+      self.subscriptions = clientOld.subscriptions;
+      self.presences = clientOld.presences;
+
+      self._persist();
```

Comment:
There will be a new createdAt, id properties.  We could decide *not* to store this updated data, and live with the original values.  Any *additional* storeData() calls will persist the these new properties to persistence.

So the call to expire() can be taken out also.